src/Commands/Customers/CustomersGroups.php

```diff
@@ -35,6 +35,11 @@
  */
 final class CustomersGroups extends Command
 {
+    public const SUCCESS = 0;
```

Comment:
I though this constants were available with this symfony version. [But they are not](https://github.com/symfony/console/blob/3.4/Command/Command.php).
It's only [for later versions](https://github.com/symfony/console/blob/15c96194f32e1b1aa30d1b302c71c5f83fd4dea9/Command/Command.php#L32).

It's nice that you added it !
(We'll move them to the base command later.)

Reply:
Yes !
I noticed that in our version they don't exist, that's why I added them in my class. But I think for the sake of reusability we should put them in the Command class.